src/Controller/GeocoderController.php

```diff
@@ -1,8 +1,4 @@
 <?php
-/**
- * @file
- * Contains \Drupal\geocoder_autocomplete\Controller\GeocoderController.
- */
```

Comment:
phpcs (in my phpstorm) says that is missing doc comment, if we remove these comments lines